api/admin/templates.py

```diff
@@ -18,11 +21,21 @@
         email: \"{{ email }}\",
         roles: [{% for role in roles %}{"role": \"{{role.role}}\"{% if role.library %}, "library": \"{{role.library.short_name}}\"{% endif %} },{% endfor %}]
     });
+    const elementsToRemove = document.getElementsByClassName("error");
+    while(elementsToRemove.length > 0){
+        elementsToRemove[0].parentNode.removeChild(elementsToRemove[0]);
+    }
+  } catch (e) {
+    document.getElementById("error1").innerHTML = "We're having trouble displaying this page."
+    document.getElementById("error2").innerHTML = "Contact your administrator, and ask them to check the console for more information."
+    console.warn("The CSS and/or JavaScript files for this page could not be found. Try running `npm install` in the api/admin directory of the circulation repo.")
+  }
```

Comment:
Is the exception `e` itself useful here, or are you pretty sure this will only ever happen due to missing files? If it could be a different problem, it would be good to log `e` to the console as well.

Reply:
I added a console.error for `e`!